plugins/README.md

```diff
@@ -0,0 +1,27 @@
+# STEPS
+
+Below steps need to be run from the root repository
+
+- npx lerna create <package-name> -y
+- npx lerna add <npm-page-name> // make sure we have the same version of the npm-package-name in each package by calling lerna add from the root of the repository. ex: npx lerna add aws-sdk
+- npx lerna add <package-name> // this links <package-name> to all packages ex: npx lerna add common
```

Comment:
I think the above step can be mentioned after this, as trying the same command will show the version.

Reply:
ok cool